lint-configs/symlink-cpp-lint-configs.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Exit on any error
+set -e
+
+# Error on undefined variable
+set -u
+
+repo_dir="$(git rev-parse --show-toplevel)"
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Symlinks the given config file to the repo's root.
+#
+# @param $1 Path to the config file in the repo.
+function symlink_config () {
+    config_file_path="$1"
+
+    repo_dir="$(git rev-parse --show-toplevel)"
+    repo_relative_config_file_path="$(realpath --relative-to="$repo_dir" "$config_file_path")"
+
+    src_path="${repo_dir}/${repo_relative_config_file_path}"
+    dst_path="${repo_dir}/$(basename "$config_file_path")"
+
+    # NOTE: `-e` will return false if the file is a broken symlink, so that's why we also check
+    # `-L`.
+    if [ ! -e "$dst_path" ] && [ ! -L "$dst_path" ]; then
+        ln -s "$repo_relative_config_file_path" "$dst_path"
```

Comment:
Is there any output when this works? If not, maybe useful to add:
echo "Symlinked '$dst_path' to '$config_file_path'"